src/index.ts

```diff
@@ -93,3 +84,24 @@ if (!isPost) {
 } else {
   runPost()
 }
+
+function generateTraceId(): string {
```

Comment:
This is coming from otel.
https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-core/src/platform/node/RandomIdGenerator.ts#L35